src/Pages/Homepage.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import fbIcon from '../static/images/icons8-facebook-48.png';
+import twitterIcon from '../static/images/icons8-twitter-48.png';
+import '../static/css/homepage.css';
+
+
+function Homepage() {
+  return (
+    <>
+      <header>
+        <div className="asd">
+          <h1 className="title">The Template Project</h1>
+          <p className="caption">
+            A one stop shop for beautifully designed ReactJS templates
+          </p>
+        </div>
+        <div className="social-icons-container">
```

Comment:
Target blank, is probably useful; https://www.w3schools.com/tags/att_a_target.asp

Reply:
done, also added  `rel="noopener noreferrer"` to the `a` to protect against this vulnerability (although twitter and fb are pretty trustworthy) https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/